src/pal/inc/pal.h

```diff
@@ -3299,6 +3299,13 @@ PALAPI
 LPCVOID
 PAL_GetCoreClrModuleBase();
 
+PALAPI
+DWORD
+PAL_GetModuleProcAddress(
```

Comment:
Do you still need this? I think this is just a leftover from your experimental code, right?